### ISSUE TITLE
Range iteration bug

### ISSUE THREAD

**Issue by pshriwise** [scopatz/pymoab#13]
When trying to iterate over a Range object like so 

```
verts = mb.create_vertices(coords)
for vert in verts:
    print vert
```

it causes an infinite loop. It seems some of the necessary iteration information isn't there. I took a crack at trying to fix this and didn't come up with much. Any thoughts @scopatz?

**Comment by vijaysm** [comment]
Can you check what is verts.size() and verts.psize() ? I don't see an iterator exposed for range in pymoab. So can you just loop over it like a list without it ? That piece of the code looks strange.

**Comment by pshriwise** [comment]
size() and psize() appear to be the same. There is an iterator for Range that I was thinking about exposing, but it seems to be of non standard type and I don't know if cython will pick it up like it would an iterator for a standard vector. I think we only exposed as much as we needed for the Range class.

**Comment by pshriwise** [comment]
Fixed in #14
